src/js/_enqueues/lib/nav-menu.js

```diff
@@ -589,8 +589,11 @@
 		},
 
 		initPreviewing : function() {
+
+			const menuToEdit = $( '#menu-to-edit' );
```

Comment:
Make it Var instead of Const. Also remove extra line from top and bottom.

Reply:
- Resolved in https://github.com/WordPress/wordpress-develop/pull/6389/commits/24598ed85737201ee50112246c4c0b6da3458029 🚀 .